### ISSUE TITLE
Doesn't work with `<<fork>>` option

### ISSUE THREAD

**Issue by majcn** [ChappIO/gatsby-remark-mermaid#13]
example: https://mermaid-js.github.io/mermaid/#/stateDiagram?id=forks

workaround: replaced `<` with `&lt;`

**Comment by DanielFallon** [comment]
ran into a similar issue using `<path>` to designate a cli parameter in a sequence diagram.

Thanks for the workaround :)

## Debugging

as for my experience with the bug, I found that the generated `index.html` and `<svg>` element contained the following code:

```svg
<text x="279" y="346" text-anchor="middle" dominant-baseline="middle" alignment-baseline="middle" class="messageText" dy="1em" style="font-family: &quot;trebuchet ms&quot;, verdana, arial, sans-serif; font-size: 16px; font-weight: 400;">
    &lt;path&gt;&lt;/path&gt;
</text>
```

The tags are being escaped eventually, so tag closing is happening during page generation, and elsewhere (say, during rendering time in the browser.) 


Additionally, when multiple were used, the associated closing tags were shoved into the last <text> element that existed with an unclosed tag. Not sure how it chooses which `<text>` element to add the closing tags to, but they all seemed to go to a single place.

**Comment by remcohaszing** [comment]
I think this was fixed in version 3.0.0. If not, please report an issue to [`remark-mermaidjs`](https://github.com/remcohaszing/remark-mermaidjs).
